components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -719,7 +723,6 @@ export class GitpodServerImpl<Client extends GitpodClient, Server extends Gitpod
         if (!ws) {
             throw new Error(`No workspace with id '${id}' found.`);
         }
-        await this.guardAccess({ kind: "workspace", subject: ws }, "get");
```

Comment:
Thanks for removing this duplicate access guard. 👍

The only place that relied solely on this access guard was `resolvePlugins` -- could you please add an access guard to `resolvePlugins`? 🙏

Reply:
This is still unresolved, as `resolvePlugins` needs an access guard:

https://github.com/gitpod-io/gitpod/blob/5b6fec801c648c0cc39d167d3f880cf46508c7ad/components/server/src/workspace/gitpod-server-impl.ts#L1678-L1684